sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -1088,6 +1145,21 @@ def create_operation(self,
         transform_proto.spec.payload, parameter_type)
     return creator(self, transform_id, transform_proto, payload, consumers)
 
+  def extract_timers_info(self):
```

Comment:
I would populate output_stream here as well rather than above.

Reply:
We can only populate output_stream when processing bundle since instruction_id is required.

Reply:
Ack, expanding the whole diff I see that this is happening in different methods now (in which case here two separate maps, as you had originally, might be preferable). But not a big deal.

Reply:
The keys of maps should still be the same though(tuple of (transform_id, timer_family_id)). That's why I make the value as a map{coder_impl, output_stream}